packit/base_git.py

```diff
@@ -371,7 +371,13 @@ def set_specfile_content(self, specfile: SpecFile, version: str, comment: str):
         self.specfile.set_version(this_version)
         self.specfile.save()
         rpmdev_bumpspec(self.absolute_specfile_path, comment=comment, version=version)
-        self.specfile._update_data()  # refresh the spec after we changed it
+        # refresh the spec after we changed it
+        if hasattr(self.specfile, "update"):
+            # new rebase-helper
+            self.specfile.update()
+        else:
+            # old rebase-helper
+            self.specfile._update_data()
```

Comment:
This sounds like we can create some custom wrapper around rebase helper with all these fixes in one place.

(no need to fix here)

Reply:
yup, good idea: we could inherit from rebasehelper's classes and gather the fixes in there

Reply:
can you create an issue for that please?

Reply:
https://github.com/packit-service/packit/issues/560